scripts/install_third_party_libs_test.py

```
@@ -747,6 +756,91 @@ def mock_print(arg: str) -> None:
+
+        with self.exists_false_swap, self.is_x64_architecture_true_swap:
+            with os_name_swap, self.assertRaisesRegex(
+                Exception, 'Unsupported OS'
+            ):
+                install_third_party_libs.install_playwright_node()
+
+    def test_install_playwright_node_with_non_x64_architecture(
+        self,
+    ) -> None:
+        with self.download_swap, self.rename_swap, self.exists_false_swap:
+            with self.is_x64_architecture_false_swap:
```

Ruff (0.15.14)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)